Makefile

```diff
@@ -138,11 +140,20 @@ sdk-python: api/swagger.yml  ## Generate SDK for Python client - openapi generat
 		--package-name lakefs_sdk \
 		--http-user-agent "lakefs-python-sdk/$(PACKAGE_VERSION)" \
 		--git-user-id treeverse --git-repo-id lakeFS \
-		--additional-properties=infoName=Treeverse,infoEmail=services@treeverse.io,packageVersion=$(PACKAGE_VERSION),packageName=lakefs_sdk,packageVersion=$(PACKAGE_VERSION),projectName=lakefs-sdk,packageUrl=https://github.com/treeverse/lakeFS/tree/master/clients/python \
+		--additional-properties=infoName=Treeverse,infoEmail=services@treeverse.io,packageVersion=$(PACKAGE_VERSION),projectName=lakefs-sdk,packageUrl=https://github.com/treeverse/lakeFS/tree/master/clients/python \
 		-c /mnt/clients/python/python-codegen-config.yaml \
 		-o /mnt/clients/python \
 		--ignore-file-override /mnt/clients/python/.openapi-generator-ignore
 
+sdk-rust: api/swagger.yml  ## Generate SDK for Rust client - openapi generator version 7.1.0
+	rm -rf clients/rust
+	mkdir -p clients/rust
+	$(OPENAPI_RUST_GENERATOR) generate \
+		-i /mnt/api/swagger.yml \
+		-g rust \
+		--additional-properties=infoName=Treeverse,infoEmail=services@treeverse.io,packageName=lakefs_sdk,packageVersion=$(PACKAGE_VERSION),packageUrl=https://github.com/treeverse/lakeFS/tree/master/clients/rust \
+		-o /mnt/clients/rust
+
```

Comment:
This generated code is not rustfmt'ed.

We need either to:
* Add `--enable-post-process-file` here (and verify that it also works with the "rust" generator, because maybe it only works with "rust-server"), OR
* Add a "rustfmt" / "cargo fmt" command here to ensure that it's formatted (and then we will need to do something smarter about validation below!)

Reply:
Not sure why we need to go to these lengths - it's auto generated code - if it compiles I don't care if it doesn't pass fmt
You can say the same for the python client which doesn't adhere to pep8 but we still don't perform any post processing

Reply:
After further checking, rustfmt takes a long time to run so I'm not sure it will be a good idea to add it to any of our makefile targets (build clients / checks-validator). Will merge as is for now and handle this later when this becomes an issue